examples/jsm/tsl/display/AfterImageNode.js

```diff
@@ -50,6 +51,10 @@ class AfterImageNode extends TempNode {
 
 		const { renderer } = frame;
 
+		_rendererState = PostProcessingUtils.resetRendererState( renderer, _rendererState );
```

Comment:
Regarding the name: I would expect something like `saveRendererState()` since what we doing here is essentially saving a snapshot of the current render state which is later restored. So how about?
```js
PostProcessingUtils.saveRendererState( renderer, _rendererState );
PostProcessingUtils.restoreRendererState( renderer, _rendererState );
```